frontend/src/components/QuestionInput/QuestionInput.tsx

```diff
@@ -186,7 +187,7 @@ export const QuestionInput = ({ onSend, disabled, placeholder, clearOnSend, extr
             return;
         }
 
-        onSend(question, fileBlobUrl);
+        onSend(question, fileBlobUrl, agent);
```

Comment:
@ramirezmorac2, it appears that the agent value is always empty. I don't see the point of sending this from here